manual/conf.py

```diff
@@ -15,7 +15,21 @@
     "jupyter_sphinx",
 ]
 
-html_theme = "sphinx_rtd_theme"
+html_theme = "sphinx_book_theme"
+
+pygments_style = "pastie"
+
+html_theme = "sphinx_book_theme"
+
+html_theme_options = {
+    "repository_url": "https://github.com/CQCL/tket",
```

Comment:
What does this do? Do we want it to be this repo?

Reply:
This tells sphinx which repo to link to in the button at the top of the page. I decided to make it the tket repo as raising issues there is preferred. Although I could link to the pytket repo instead.

Reply:
OK that makes sense, thanks.